package.json

```diff
@@ -301,7 +300,7 @@
     "react-dom@^18.2.0": "patch:react-dom@npm%3A18.2.0#./.yarn/patches/react-dom-npm-18.2.0-dd675bca1c.patch",
     "react-dev-utils/fork-ts-checker-webpack-plugin": "^6.5.3",
     "p-queue": "^6.6.2",
-    "web3-eth-contract": "1.7.4"
+    "web3": "4.2.1-dev.a0d6730.0"
```

Comment:
Note how we pin the resolution, so that the transient packages use the pinned, lighter version. Effectively unused however:

```shell
❯ yarn why web3
└─ @celo/contractkit@npm:1.5.2
   └─ web3@npm:4.2.1-dev.a0d6730.0 (via npm:4.2.1-dev.a0d6730.0)
```

```shell
❯ yarn why @celo/contractkit
├─ @ledgerhq/live-common@npm:21.36.5
│  └─ @celo/contractkit@npm:1.5.2 (via npm:^1.5.2)
│
└─ @ledgerhq/live-common@npm:21.36.5 [8a243]
   └─ @celo/contractkit@npm:1.5.2 (via npm:^1.5.2)
```